bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -117,6 +123,9 @@ public Builder withProjectId(String projectId) {
 
     /**
      * {@inheritDoc}
+     *
```

Comment:
please move these comment to a different PR.

Reply:
done.